content/hi/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Windows containers in Kubernetes
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+Windows एप्लिकेशन कई संगठनों में चलने वाली सेवाओं और अनुप्रयोगों के एक बड़े हिस्से का गठन करते हैं। [Windows कंटेनर](https://aka.ms/windowscontainers) प्रक्रियाओं और पैकेज निर्भरता को समाहित करने का एक तरीका प्रदान करते हैं, जिससे DevOps प्रथाओं का उपयोग करना और विंडोज अनुप्रयोगों के लिए क्लाउड नेटिव पैटर्न का पालन करना आसान हो जाता है। 
+
+विंडोज-आधारित अनुप्रयोगों और लिनक्स-आधारित अनुप्रयोगों में निवेश वाले संगठनों को अपने कार्यभार को प्रबंधित करने के लिए अलग ऑर्केस्ट्रेटर की तलाश करने की आवश्यकता नहीं है, जिससे ऑपरेटिंग सिस्टम की परवाह किए बिना, उनके परिनियोजन में परिचालन क्षमता में वृद्धि होती है।
```

Comment:
We either need to Windows or विंडोज across the blog as a standard. Having both will confuse folks.

Reply:
Ditto with लिनक्स or Linux.

Reply:
Yup, I'm aware of this, actually the translator tool does this inconsistency. Will clear once and for all.